### DISCUSSION TITLE
Adding to start menu without installing

### DISCUSSION THREAD

**Discussion by benfulton** [microsoft/wslg#601] [Q&A] [answered]
I've read that when you install apps with the distro package manager they show up on the Windows start menu. I tend to use Spack rather than the default package manager though. How would I add apps to the Windows start menu manually?

**Comment by JesseMitchell-Reiss** [comment, marked as answer]
If you install an app and it is not in the start menu  but you want it to be then you can follow these steps to create one:
%appdata%\Microsoft\Windows\Start Menu\Programs > context menu (right click) > new > shortcut
shortcut location: %wslg location (default: C:\Windows\System32\wslg.exe)% -d %distro% %command to start app%
